test/test_tsubaiso_sdk.rb

```diff
@@ -374,6 +375,14 @@ def test_show_reimbursement
     @api.destroy_reimbursement(reimbursement[:json][:id])
   end
 
+  def test_show_reimbursement_reason_master
```

Comment:
Since we didn't have create API request for xxxxReasonMaster API, I think it's OK to make test based on our fixtures. 
@reim_reason_mst_1[:id] got from our fixtures..